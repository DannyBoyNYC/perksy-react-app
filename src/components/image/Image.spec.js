const React = require('react');

const Image = require('./index').default;

describe('<Image.* />', () => {
  let component;
  const requiredProps = {
    altText: 'Foo image',
    height: 100,
    src: '/foo.jpg'
  };

  describe('<Image.Lazy />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(
        <Image.Lazy {...requiredProps} />,
        'Image'
      );
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
