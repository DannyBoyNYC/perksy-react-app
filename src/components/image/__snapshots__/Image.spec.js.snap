// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image.* /> <Image.Lazy /> should render correctly with all required props 1`] = `
<LazyLoad
  className="Image-root-0-1-1"
  height={100}
  offset={100}
  once={false}
  overflow={false}
  resize={false}
  scroll={true}
  unmountIfInvisible={false}
>
  <img
    alt="Foo image"
    className="Image-image-0-1-2"
    height={100}
    src="/foo.jpg"
  />
</LazyLoad>
`;
